.gitignore

```diff
@@ -4,3 +4,6 @@ generated/
 /**/output*/*
 !**/output-esm/package.json
 !**/output-cjs/package.json
+uniswap-oracle.iml
+artifacts/
+cache
```

Comment:
What are these files and why are they being ignored?

Reply:
hardhat compiler output

Reply:
```suggestion
uniswap-oracle.iml
artifacts/
cache/
```
Preferable?